src/Modules/AuthenticationModule/components/ResetPass/ResetPass.tsx

```diff
@@ -1,7 +1,256 @@
+  const { showToast } = useToast();
 
+  const [showPassword, setShowPassword] = React.useState(false);
 
-export default function ResetPass() {
+  const handleClickShowPassword = () => setShowPassword((show) => !show);
+
+  const navigate = useNavigate();
+  let {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<Input>();
+  const onSubmit: SubmitHandler<Input> = async (data) => {
+    try {
+      const response = await axios.post(
+        "https://upskilling-egypt.com:3000/api/v0/admin/users/reset-password",
+        data
+      );
+      showToast("success", response.data.message);
+      navigate("/login");
+
+    } catch (error: any) {
+      const err = getErrorMessage(error);
+      showToast("error", err);    }
+  };
   return (
-    <div>ResetPass</div>
-  )
+    <>
+      <Grid
+        container
+        display="flex"
+        justifyContent="center"
+        
+      >
+        <Grid item xs={12} md={6} sm={12} sx={{ ml: "50px" }}>
+          <Typography sx={{ m: "15px", color: "#152C5B", fontWeight: 700 }}>
+            <Typography
+              sx={{
+                display: "inline-block",
+                color: "#3252DF",
+                fontWeight: 700,
+              }}
+            >
+              Stay
+            </Typography>
+            cation.
+          </Typography>
+          <Box sx={{ my: "25px" }}>
+            <Typography variant="h5" sx={{ ml: "100px", fontWeight: 700 }}>
+              Reset Password
+            </Typography>
+            <Typography sx={{ ml: "100px" }}>
+              If you already have an account register
+              <Typography>
+                You can
+                <Link
+                  style={{
+                    color: "#ff0000",
+                    textDecoration: "none",
+                    marginInline: "5px",
+                  }}
+                  to="/"
+                >
+                  Login here !
+                </Link>
+              </Typography>
+            </Typography>
+          </Box>
+          <Box
+            onSubmit={handleSubmit(onSubmit)}
+            component={"form"}
+            sx={{
+              width: "60%",
+              mx: "100px",
+            }}
+          >
+            <label style={{ color: "#3252DF" }}>Email</label>
+            <TextField
+              focused={false}
+              sx={{
+                width: "100%",
+                backgroundColor: "#F5F6F8",
+                my: "7px",
+              }}
+              placeholder="Enter your Email"
+              variant="filled"
+              size="small"
+              {...register("email", {
+                required: "Email is required",
+              })}
+            />
+            {errors.email && (
+              <Alert variant="filled" severity="error">
+                {errors.email.message}
+              </Alert>
+            )}
+            <label style={{ color: "#3252DF" }}>OTP</label>
+            <TextField
```

Comment:
There are inconsistencies in writing our code and our appearance in our application. In some components, we are using the TextField (which have label, input,helperText (can be used as an error message)). and in other places, we use different elements. We should stick to one and maybe make a custom component for us and use it every where to have the same design across our application.